### PR TITLE
chore: Fix and enhance tax and invoice information in wizard step 1

### LINKED ISSUE #7957: Fix and enhance tax and invoice information in wizard step 1 and invoice PDF
1. Always display button "Add tax and invoice information" no matter if it is a paid event or free event.
Screenshot from 2021-10-19 13-38-32
2. Change "Add tax information" to "Add invoice and tax information"
3. Below "Registered Organisation" add fields for 
  * Phone
  * Email
Screenshot from 2021-10-19 13-36-21
4. Always display info in invoices from this form. Do not include account info of owner anymore. If there is no info keep the invoice empty.

### PR DESCRIPTION
**Pull request by iamdevvalecha**
Fixes #7957 

#### Short description of what this resolves:
1.display button "Add tax and invoice information" no matter if it is a paid event or free event.
2.Change "Add tax information" to "Add invoice and tax information"
3.Below "Registered Organisation" add fields for
- Phone
- Email
## part 1 & 2
Screenshot from 2021-10-20 10-30-50
Screenshot from 2021-10-20 10-30-53
## part 3
Screenshot from 2021-10-20 10-31-06
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mariobehling** [comment]
@pc-beast I think we need a server PR for the Email and Phone data as well, right?

**Comment by pc-beast** [comment]
> @pc-beast I think we need a server PR for the Email and Phone data as well, right?

Yes, @iamdevvalecha please do required server changes too. Also please test the complete functionality yourself before filing a complete PR.

**Comment by mariobehling** [comment]
Once the tax information is filled in, where do you display the invoice/tax info box?

Screenshot from 2021-10-04 19-27-31

I suggest to move it directly below the blue buttons or do you have another suggestions?

Screenshot from 2021-10-21 14-45-51

**Review by codedsun — COMMENTED** [review]
Fix the failing test

File: app/mixins/event-wizard.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
 
       // model.tax is set in basic-detail step to workaround issue #4385
       let tax = this.model.tax || data.tax;
-      if (tax && tax.name) {
+      if (tax) {
```
Comment (maze-runnar) [inline app/mixins/event-wizard.js:101]: don't do this

Comment (progmatic-99) [reply app/mixins/event-wizard.js:101]: Is there any specific reason??

**Review by mariobehling — APPROVED** [review]
Thank you! Good job. Let's test this further on the test system.

File: app/mixins/event-wizard.js (hunk 1 above)
Comment (maze-runnar) [reply app/mixins/event-wizard.js:101]: Yes there is, when something in tax will break we will see.

**Comment by mariobehling** [comment]
I pressed merge too quickly. As you mentioned the tests are still failing. So, I am reverting this. Please try to make the test pass.

**Pull request merged**